elements/pfe-jump-links/src/pfe-jump-links.js

```diff
@@ -93,18 +93,20 @@ class PfeJumpLinksNav extends PFElement {
         if (this.getAttribute("sr-text")) {
           this.shadowRoot.querySelector("nav").prepend(div);
         }
-
+        if (this.hasAttribute("horizontal")) {
+          return;
+        }
         let html = "";
-        if (this.querySelector(".pfe-jump-links-nav--heading")) {
-          html = this.querySelector(".pfe-jump-links-nav--heading").cloneNode(
-            true
-          );
+        if ("[slot='pfe-jump-links-nav--heading']") {
+          html = this.querySelector("[slot='pfe-jump-links-nav--heading']")
+            .innerHTML;
+        } else {
+          html = "Jump to section";
         }
         if (!this.hasAttribute("horizontal")) {
```

Comment:
Do you still need this since you added the escape on line 96?

Reply:
Yes, I think so. No harm in leaving it because it would handle any edge case like no slot or an undefined value.

Reply:
I don't think it will.  It escapes completely on line 96 so you would never hit this point if the horizontal attribute exists.